### PR TITLE
Cleanup cli inspect code

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

File: api/client/inspect.go [hunk 1]
```diff
@@ -46,112 +44,125 @@ func (cli *DockerCli) CmdInspect(args ...string) error {
...
-				} else {
-					fmt.Fprintf(cli.err, "%s", err)
+		switch *inspectType {
+		case "container":
+			obj, statusCode, err = cli.getContainer(name)
+			if err != nil {
+				if statusCode == http.StatusNotFound {
+					fmt.Fprintf(cli.err, "No such container: %s\n", name)
+					status = 1
+					continue
```
Comment (LK4D4) [inline api/client/inspect.go]: This is continue for `switch`? What does it mean?

Comment (cpuguy83) [inline api/client/inspect.go]: oh boo, does switch take `continue`?

**Comment by LK4D4** [comment]
I dunno, there is more code lines and it became harder to read for me because of all this `continue`.

File: api/client/inspect.go (hunk 1 above)
Comment (LK4D4) [inline api/client/inspect.go]: Have no idea. That's why I'm asking.

**Comment by cpuguy83** [comment]
Updated.
It's a few extra lines in the actual code (I think +4), but I find this more readable.
WDYT?

**Comment by LK4D4** [comment]
@cpuguy83 Now looks better for me. I can read through code and understand all of it.

**Comment by cpuguy83** [comment]
Yeah, but there's some issues :(
This is a real mess.

**Comment by LK4D4** [comment]
@cpuguy83 indeed. At least we have tests :)

**Pull request closed without merging**